app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -243,6 +244,21 @@ class HomeActivityTest {
     assertThat(screenName).isEqualTo(ScreenName.HOME_ACTIVITY)
   }
 
+  @Test
+  fun testHomeActivity_createIntent_verifyProfileIdInIntent() {
+    val profileId = createHomeActivityIntent(internalProfileId)
+      .extractCurrentUserProfileId()
+
+    assertThat(profileId.internalId).isEqualTo(internalProfileId)
+  }
+
+  @Test
+  fun testActivity_createIntent_verifyCurrentUserProfileId() {
```

Comment:
Isn't this & the test above verifying the exact same thing? Why do we need two versions of it?

Reply:
Yeah updated to remove one of them.